deploy/kubernetes/releases/csi-digitalocean-v1.0.4.yaml

```diff
@@ -0,0 +1,526 @@
+# Copyright 2018 DigitalOcean
```

Comment:
Should we bump this copyright date in the template?

Reply:
Yeah, I guess we should do that. How about doing one big update in a dedicated, follow-up PR though? Or would you prefer updating just the template in this one?

Reply:
Yeah, I don't have strong feelings on it.  It's open source software; worst case its copyright expires and it falls into the public domain.

Reply:
Done in #272.